micro_sam/sam_annotator/_widgets.py

```diff
@@ -736,7 +736,9 @@ def segment_slice(viewer: "napari.viewer.Viewer") -> None:
         return None
 
     shape = viewer.layers["current_object"].data.shape[1:]
-    position = viewer.cursor.position
+
+    position_world = viewer.dims.point
+    position = viewer.layers["point_prompts"].world_to_data(position_world)
     z = int(position[0])
 
     point_prompts = vutil.point_layer_to_prompts(viewer.layers["point_prompts"], z)
```

Comment:
No, see my comment below. in `point_layer_to_prompts` everything is now in the image layer data coordinates, because the annotation layers have the same scale as the image layer. We had to transform `viewer.dims.point` because that is in world (scaled or canvas) coordinates.